modules/ROOT/edits/install-self-managed.adoc

```diff
@@ -0,0 +1,259 @@
+| 443 | HTTPS | * US control plane: worker-cloud-helm-prod.s3.amazonaws.com
+* EU control plane: worker-cloud-helm-prod-eu-rt.s3.amazonaws.com worker-cloud-helm-prod-eu-rt.s3.eu-central-1.amazonaws.com | Runtime Fabric version repository. The Runtime Fabric installation uses software from this repository during installation and upgrades.
+| 443 | HTTPS | * US control plane: exchange2-asset-manager-kprod.s3.amazonaws.com
+* EU control plane: exchange2-asset-manager-kprod-eu.s3.amazonaws.com exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com |Anypoint Exchange for application assets.
+| 443 | HTTPS | * US control plane: rtf-runtime-registry.kprod.msap.io
+* EU control plane: rtf-runtime-registry.kprod-eu.msap.io | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: prod-us-east-1-starport-layer-bucket.s3.amazonaws.com prod-us-east-1-starport-layer-bucket.s3.us-east-1.amazonaws.com
+* EU control plane: prod-eu-central-1-starport-layer-bucket.s3.amazonaws.com prod-eu-central-1-starport-layer-bucket.s3.eu-central-1.amazonaws.com | Runtime Fabric Docker image delivery.
+| 443 | HTTPS | * US control plane: runtime-fabric.s3.amazonaws.com
+* EU control plane: runtime-fabric-eu.s3.amazonaws.com | Runtime Fabric Docker repository.
+| 443 | HTTPS | * US control plane: configuration-resolver.prod.cloudhub.io
+* EU control plane: configuration-resolver.prod-eu.msap.io | Anypoint Configuration Resolver.
+|===
+
+=== Certificate Configuration
+
+To allow different endpoints to use mutual TLS authentication to establish a connection, you must configure SSL passthrough to allow the following certificates:
+
+[%header,cols="2*a"]
+.Cerfificate configuration for Self-Managed Kubernetes
+|===
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io +
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Step 2: Create a Runtime Fabric using Runtime Manager
+
+[IMPORTANT]
+====
+The procedures in this section should be performed by a MuleSoft organization administrator.
+====
+
+To install Runtime Fabric on Self-Managed Kubernetes, first create a Runtime Fabric using Runtime Manager. This is required to obtain the activation data which is needed during installation.
+
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Accept*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section.
```

Comment:
All UI labels referred to in context of the audience looking at the UI should be formatted in bold:
```suggestion
Runtime Manager creates the Runtime Fabric and displays the *Activation State* page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for future use.
```
Do not create intertopic dependencies by referring to "next" or "previous" topics, sections, or other doc units.